code/game/objects/structures/crates_lockers/closets/secure/engineering.dm

```diff
@@ -119,6 +117,10 @@
 	new /obj/item/clothing/gloves/white(src)
 	new /obj/item/clothing/under/whites(src)
 	new /obj/item/clothing/head/white_dress(src)
+	new /obj/item/tool/shovel/etool(src)
+	new /obj/item/squad_beacon(src)
+	new /obj/item/binoculars/tactical/range(src)
```

Comment:
A) why
B) Document this better in the CL

Reply:
mortar assistance, supply management with the RO

Reply:
rangefinders are for mortar only explain why this change exits

Reply:
I believe ST are all around supply and supplemental roles, and they arent very clearly defined.

Reply:
ST manage req and shiside engineering work(groundside FOB if no engies) theres no reason for them to have these but to encourage combat STs (and god save me if we have captain deploying argument No. 4567 all over again)

Reply:
lemme fix that then